jackson-jq/src/main/java/net/thisptr/jackson/jq/Scope.java

```diff
@@ -1,41 +1,37 @@
 package net.thisptr.jackson.jq;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Enumeration;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
-import java.util.ServiceLoader;
-import java.util.TreeMap;
-
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.MappingIterator;
 import com.fasterxml.jackson.databind.ObjectMapper;
-
 import net.thisptr.jackson.jq.exception.JsonQueryException;
 import net.thisptr.jackson.jq.internal.BuiltinFunction;
 import net.thisptr.jackson.jq.internal.JsonQueryFunction;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.*;
+import java.util.Map.Entry;
+
 public class Scope {
 	private static final ObjectMapper DEFAULT_MAPPER = new ObjectMapper();
 
+	public static final String JQ_JSON = "net/thisptr/jackson/jq/jq.json";
+
+	@Deprecated
 	private static final class RootScopeHolder {
+		@Deprecated
 		public static final Scope INSTANCE = new Scope(null);
 		static {
 			try {
-				INSTANCE.loadBuiltinFunctions();
-				INSTANCE.loadMacros();
+				final ClassLoader classLoader = Optional.ofNullable(Thread.currentThread().getContextClassLoader())
```

Comment:
This maintains the old behaviour of using the context class loader and falling back to the instance class loader.